README.md

```diff
@@ -23,7 +23,34 @@ A WIP Rust implementation of [Messaging Layer Security](https://github.com/mlswg
 
 ### Dependencies
 
-OpenMLS relies on [EverCrypt](https://github.com/project-everest/hacl-star/tree/master/providers/evercrypt), a high-performance, cross-platform, formally verified modern cryptographic provider through [EverCrypt Rust bindings](https://crates.io/crates/evercrypt).
+#### Cryptography
+
+OpenMLS does not implement its own cryptographic primitives.
+Instead, it relies on existing implementations of the cryptographic primitives used.
+There are two different cryptography backends supported right now.
+
+##### Rust Crypto
+
+This is the default cryptographic backend, using [hkdf], [sha2], [p256], [p384], [x25519-dalek-ng], [ed25519-dalek] [chacha20poly1305], [aes-gcm].
+
+
+##### Evercrypt
+
+[EverCrypt] provides a formally verified cryptographic backend through the through [EverCrypt Rust bindings] to OpenMLS.
+The default features have to disabled and the `evercrypt-backend` feature has to be enabled.
+
+```
+cargo build --features="evercrypt-backend" --no-default-features
+```
+
+Note that using this provider restricts OpenMLS usage to the following platforms right now
+* Linux x64
+* Linux Arm64
+* MacOS x64
+* Windows x64
+
+⚠️ Note that using this feature tests don't work out of the box.
+You need to change the `openmls` dev-dependency in the `Cargo.toml` to use the evercrypt backend as well.
```

Comment:
Right, that's not possible right now because the tests require the openmls with test-utils. But the general way this is handled will change again with the crypto object and I think we should be able to run with both backends then.